src/views/mixed-content/index.js

```diff
@@ -10,10 +18,17 @@ import React from 'react';
  * @return {ReactNode} Mixed Content View
  */
 const MixedContent = () => {
-  // TO-DO(ernestognw): To fill with mixed-content data graphs
   return (
     <div data-testid="mixed-content">
-      <p>Mixed Content</p>
+      <h1>Mixed Content</h1>
```

Comment:
Material-UI has its own Box, Title, or Typography components. In the future, would be better if we use them to follow the UI guidelines.

Same for others

Reply:
Done